libphal/exception.H

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <exception>
+
+namespace libphal
```

Comment:
should we use libphal or phal? lib name for namespace? unless there is a clash

Reply:
since this is phal library api's would prefer to usethis name space. phal  name space is commonly used in BMC apps.